lib/fortnox/api/repositories/order.rb

```diff
@@ -0,0 +1,33 @@
+require "fortnox/api/repositories/base"
+require "fortnox/api/models/order"
+
+module Fortnox
+  module API
+    module Repository
+      class Order < Fortnox::API::Repository::Base
+
+        CONFIGURATION = Fortnox::API::Repository::Base::Options.new(
```

Comment:
As I wrote above, maybe something for a separate PR. This concerns all `Repositories`, not just `Order`.